buildenv/Build_Instructions_V9.md

```diff
@@ -302,8 +302,99 @@ OpenJDK  - 437530c based on jdk-9+181)
 ## Windows
 :ledger:
 
-:construction:
-We haven't created a full build process for Windows yet? Watch this space!
+The following instructions guide you through the process of building a Windows OpenJDK V9 binary that contains Eclipse OpenJ9. This process can be used to build binaries for Windows 7, 8, and 10.
+
+### 1. Prepare your system
+:ledger:
+You must install a number of software dependencies to create a suitable build environment on your system.
+
+- [Cygwin](https://cygwin.com/install.html), which provides a Unix-style command line interface.
+- [Microsoft Visual Studio 2013](https://msdn.microsoft.com/en-us/library/dd831853(v=vs.120)), which is the same compiler level used by OpenJDK. Later levels of this compiler are not supported.
```

Comment:
Should we provide the actual download link: https://my.visualstudio.com/Downloads?q=visual%20studio%202013&wt.mc_id=o~msft~vscom~older-downloads